docs/aws-copilot-fargate-vapor-mongo.md

```diff
@@ -0,0 +1,573 @@
+
+This guide illustrates how to deploy a Server Side Swift workload on AWS. The workload is a REST API for tracking a To Do List. It uses the [Vapor](https://vapor.codes/) framework to program the API methods. The methods store and retrieve data in a [MongoDB Atlas](https://www.mongodb.com/atlas/database) cloud database. The Vapor application is containerized and deployed to AWS on AWS Fargate using the [AWS Copilot](https://aws.github.io/copilot-cli/) toolchain.
+
+## Architecture
+![Architecture](../images/aws/aws-fargate-vapor-mongo.png)
+
+- Amazon API Gateway receives API requests
+- API Gateway locates your application containers in AWS Fargate through internal DNS managed by AWS Cloud Map
+- API Gateway forwards the requests to the containers
+- The containers run the Vapor framework and have methods to GET and POST items
+- Vapor stores and retrieves items in a MongoDB Atlas cloud database which runs in a MongoDB managed AWS account
+
+## Prerequisites
+To build this sample application, you need:
+
+- [AWS Account](https://console.aws.amazon.com/)
+- [MongoDB Atlas Database](https://www.mongodb.com/atlas/database)
+- [AWS Copilot](https://aws.github.io/copilot-cli/) - a command-line tool used to create containerized workloads on AWS
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/) - to compile your Swift code into a Docker image
+- [Vapor](https://vapor.codes/) - to code the REST service
+- [AWS Command Line Interface (AWS CLI)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) - install the CLI and [configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html) it with credentials to your AWS account
+
+## Step 1: Create Your Database
+If you are new to MongoDB Atlas, follow this [Getting Started Guide](https://www.mongodb.com/docs/atlas/getting-started/). You need to create the following items:
+- Atlas Account 
+- Cluster
+- Database Username / Password
+- Database
+- Collection
+
+In subsequent steps, you provide values to these items to configure the application.
+
+## Step 2: Initialize a New Vapor Project
+
+Create a folder for your project.
+
+```
+mkdir todo-app && cd todo-app
+```
+
+Initialize a Vapor project named *api*.
+
+```
+vapor new api -n
+```
+
+## Step 3: Add Project Dependencies
+Vapor initializes a *Package.swift* file for the project dependencies. Your project requires an additional library, MongoDBVapor. Add the MongoDBVapor library to the project and target dependencies of your *Package.swift* file.
+
+Your updated file should look like this:
+
+**api/Package.swift**
+```swift
+// swift-tools-version:5.6
+import PackageDescription
+
+let package = Package(
+    name: "api",
+    platforms: [
+       .macOS(.v12)
+    ],
+    dependencies: [
+        .package(url: "https://github.com/vapor/vapor", .upToNextMajor(from: "4.7.0")),
+        .package(url: "https://github.com/mongodb/mongodb-vapor", .upToNextMajor(from: "1.1.0-beta.1"))
```

Comment:
I just noticed we never tagged the GA for 1.1 🤦‍♀️ I'm going to do that this week and then we can update this to not use the beta.

Reply:
Thanks! I have submitted a change to remove the beta tag in the dependency.